modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/labeledcell/v0.2.0.json

```diff
@@ -0,0 +1,88 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/labeledcell/v0.2.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.4",
+    "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:LabeledCellShape",
+      "@type": "sh:NodeShape",
+      "comment": "Labeled cell shape definition",
+      "targetClass": "nsg:LabeledCell",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:projectName",
+              "description": "Project name",
+              "name": "Project name for data produced within the context of a project",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:brainLocation",
+              "description": "Brain location",
+              "name": "Brain location of the labeled cell.",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.1/shapes/BrainLocationShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:putativeMType",
+              "name": "Putative morphology type",
+              "description": "Indicates the putative morphology type of the labeled cell.",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.4/shapes/CellTypeOntologyTermShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:locationInSlice",
+              "name": "Coordinates of location in slice",
+              "description": "Specifies the coordinates of the location of the cell in the slice",
+              "class": "{{base}}/schemas/neurosciencegraph/commons/vector3d/v0.1.0/shapes/Vector3DShape",
```

Comment:
I suggest to replace "class" by "node". This is an error introduced in the previous version.